shared/src/test/diff/nu/SimpleUCS.mls

```diff
@@ -129,17 +116,20 @@ fun f(x, y) =
   if x is
     Some(xv) and y is Some(yv) then xv + yv
     None() and y is None() then 0
-//│ ╔══[ERROR] not exhaustive
-//│ ║  l.129: 	  if x is
-//│ ║         	     ^^^^
-//│ ║  l.130: 	    Some(xv) and y is Some(yv) then xv + yv
-//│ ║         	^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-//│ ║  l.131: 	    None() and y is None() then 0
-//│ ╙──       	^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+//│ ╔══[ERROR] The match is not exhaustive.
+//│ ║  l.117: 	    Some(xv) and y is Some(yv) then xv + yv
+//│ ║         	                 ^^^^^^^^^^^^^
+//│ ╟── The pattern at this position misses 1 case.
+//│ ║  l.117: 	    Some(xv) and y is Some(yv) then xv + yv
+//│ ║         	                 ^
+//│ ╟── [Missing Case 1/1] The case `None` is missing, which first appears here.
+//│ ║  l.118: 	    None() and y is None() then 0
+//│ ╙──       	                    ^^^^
 //│ f: (anything, anything,) -> error
 //│ Code generation encountered an error:
 //│   if expression has not been desugared
 
+// FIXME
```

Comment:
Yeah, this should be an exhaustiveness error (or at the very least a "useless case" warning for the `None() then xv * 2` part.

Reply:
This is fixed.